Dockerfile

```diff
@@ -19,20 +19,27 @@ RUN echo ${COMMIT_HASH} > ./build/COMMIT_HASH && \
     echo ${DATE} > ./build/BUILD_DATE
 
 # production stage
-FROM quay.io/mlrun/nginx:stable-alpine as production-stage
+FROM gcr.io/iguazio/nginx-unprivileged:1.21-alpine as production-stage
 
-RUN apk update && \
-	apk upgrade && \
-	rm -rf /var/cache/apk/*
+USER root
+# needs permissions to run these updates
+RUN apk update --no-cache && apk upgrade --no-cache
+USER $UID
```

Comment:
worth mentioning that you "inherit" the UID from your base image, which is 101 (add reference to the dockerfile)